foghorn-edge/templates/address-secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.